clijs/src/commands/config/set.ts

```diff
@@ -0,0 +1,48 @@
+import { Args } from "@oclif/core";
+import { BaseCommand } from "../../base.js";
+import { ConfigKeys } from "../../common/config.js";
+
+// Supported options in the config file
+const supportedOptions = new Set([
+  ConfigKeys.RpcEndpoint,
+  ConfigKeys.CometaEndpoint,
+  ConfigKeys.FaucetEndpoint,
+  ConfigKeys.PrivateKey,
+  ConfigKeys.Address,
+]);
+
+export default class ConfigSet extends BaseCommand {
+  static override description = "Set the value of a key in the config file";
+
+  static override examples = ["<%= config.bin %> <%= command.id %>"];
+
+  static args = {
+    key: Args.string({
+      description: "The key to set in the config file",
+      required: true,
+    }),
+    value: Args.string({
+      description: "The value to set for the key",
+      required: true,
+    }),
+  };
+
+  public async run(): Promise<string> {
+    const { args } = await this.parse(ConfigSet);
+    const { key, value } = args;
+
+    if (!this.configManager) {
```

Comment:
Is this possible? We initialize it in `BaseCommand` and I don't see any such check in other commands. I find it unnecessary and only confusing.